cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/lineage/field/FieldLineageInfo.java

```diff
@@ -321,6 +335,17 @@ private long computeChecksum() {
                                      operationsMap.get(input.getOrigin()), write, summary);
       }
     }
+    for (String field : droppedFields.keySet()) {
```

Comment:
If you change the `droppedFields` to a Set, here it will be exactly same logic as WriteOperations

Reply:
It's still slightly different, so I'll keep the different code anyway.